wp-content/themes/currentorg/less/current-ltw-projects.less

```diff
@@ -174,26 +239,47 @@
 			display: none;
 		}
 	}
+	@media (max-width: 770px) {
+		max-width: 97%;
+	}
 }
 .project-tags {
 	padding: 0;
 	margin: 1em 0;
 	li {
 		display: inline-block;
-		padding: 0.5em;
+		padding: 4px 6px;
 		border: 1px solid @grey-d;
 
 		& + li {
 			margin-left: 0.5em;
 		}
 	}
-	[class*=project-status] {
-		background: yellow;
+	.project-status-finalist {
+		background: @finalist;
+		color: @white;
+		a {
+			color: @white;
+		}
+	}
+	.project-status-semi-finalist {
```

Comment:
Were there specific ones in the import marked as semifinalist or winner?

Reply:
Not that I'm aware of